index.html

```diff
@@ -0,0 +1,307 @@
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gaegu:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <script defer src="src/index.js" type="module"></script>
+  </head>
+  <body>
+    <header>
+      <h1>velog</h1>
+      <div class="icons">
+        <span class="material-symbols-outlined" size="xl"> sunny </span>
+        <span class="material-symbols-outlined"> search </span>
+        <span id="newPost"><a href="write.html">새 글 작성</a></span>
+        <span id="profile">
+          <span><img src="img/profile.jpg" /></span>
+          <span class="material-symbols-outlined"> arrow_drop_down </span></span
+        >
+      </div>
+    </header>
+
+    <div class="menuBar">
+      <div class="menuBar_left">
+        <div class="navBox">
+          <nav class="navbar">
+            <span class="material-symbols-outlined"> show_chart </span>
+            <a href="#">트렌딩</a>
+          </nav>
+          <nav class="navbar">
+            <span class="material-symbols-outlined"> schedule </span>
+            <a href="#">최신</a>
+          </nav>
+        </div>
+        <div class="dropDown">
+          <div class="dropDown_clickable">
+            <span>이번 주</span>
+            <span class="material-symbols-outlined"> arrow_drop_down </span>
+          </div>
+          <div class="dropDown_content">
+            <a href="#">오늘</a>
+            <a class="dropDown_content_clicked" href="#">이번 주</a>
+            <a href="#">이번 달</a>
+            <a href="#">올해</a>
+          </div>
+        </div>
+      </div>
+      <div class="menuBar_right">
+        <span class="material-symbols-outlined"> more_vert </span>
+      </div>
+    </div>
+
+    <main>
+      <div class="card">
+        <img src="img/강아지.jpg" />
+        <div class="card_contents">
+          <h3 class="card_contents_title">힙한 강아지</h3>
+          <span class="card_contents_text"
+            >안녕하세요? 저는 강아지입니다. 저는 선글라스 끼고 드라이브하는 것을
+            제일 좋아해요. 이 사진은 날씨 좋은 날 찍은 사진입니다. 저 좀
+            귀엽죠</span
+          >
+          <div class="card_contents_info1">
+            <span>2023년 10월 25일&nbsp;·&nbsp;</span>
+            <span>33개의 댓글</span>
+          </div>
+          <div class="card_contents_info2">
+            <div class="card_contents_info2_writer">
+              <img src="img/dogprofile.jpg" />
+              <span>by HOTDOG</span>
+            </div>
+            <div class="card_contents_info2_heart">
+              <span id="heart" class="material-symbols-outlined">
+                favorite
+              </span>
+              <span>47</span>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
카드 컴포넌트 부분을 좀 더 시맨틱한 테그를 사용해보아도 좋을 것 같습니다! 
```suggestion
      <section class="card">
        <img src="img/강아지.jpg" />
        <article class="card_contents">
          <h3 class="card_contents_title">힙한 강아지</h3>
          <p class="card_contents_text"
            >안녕하세요? 저는 강아지입니다. 저는 선글라스 끼고 드라이브하는 것을
            제일 좋아해요. 이 사진은 날씨 좋은 날 찍은 사진입니다. 저 좀
            귀엽죠</p
          >
          <div class="card_contents_info1">
            <span>2023년 10월 25일&nbsp;·&nbsp;</span>
            <span>33개의 댓글</span>
          </div>
          <footer class="card_contents_info2">
            <div class="card_contents_info2_writer">
              <img src="img/dogprofile.jpg" />
              <span>by HOTDOG</span>
            </div>
            <div class="card_contents_info2_heart">
              <span id="heart" class="material-symbols-outlined">
                favorite
              </span>
              <span>47</span>
            </div>
          </footer>
        </article>
      </section>
```